ofrak_core/ofrak/service/serialization/serializers/__init__.py

```diff
@@ -0,0 +1,7 @@
+# Choosing the "Make Users Suffer" option pending resolution of
```

Comment:
Clean up this comment - no one is suffering. As you said, this is nicer for users because it resolves these warning.